src/Debug.cpp

```diff
@@ -0,0 +1,23 @@
+
+#include <OsqpEigen/Debug.hpp>
+
+// Taken from https://stackoverflow.com/a/8243866/2702753
+class NullStream : public std::ostream {
+public:
+  NullStream() : std::ostream(nullptr) {}
+  NullStream(const NullStream &) : std::ostream(nullptr) {}
+};
```

Comment:
This definition will pollute the global symbols of the osqp-eigen library with a symbol that is not in the `OsqpEigen` namespace, making it prone to collisions. Could it make sense to move this in the `OsqpEigen` namespace?